docs/Changelog.md

```diff
@@ -34,6 +34,7 @@ See the [OTP2 Migration Guide](OTP2-MigrationGuide.md) on changes to the REST AP
 - Refactor how to specify access/egress/direct/transit modes in the internal model and the Transmodel API (#3011)
 - Make agency id feed scoped (#3035)
 - Refactor kiss and ride to a more general car pickup mode (#3063)
+- Add publicCode/privateCode to the Trip class in the OTP model (#3088)
```

Comment:
Change this, refrase and write something that is meaningful for someone who look at OTP as a black box - a Product Owner.